src/utilcode/appxutil.cpp

```
@@ -861,5 +861,170 @@ namespace AppX
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_STRING_L1_1_0, WindowsGetStringRawBuffer)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_STRING_L1_1_0, WindowsCreateStringReference)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_STRING_L1_1_0, WindowsDeleteString)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_STRING_L1_1_0, WindowsGetStringLen)
+DELAY_LOADED_FUNCTION_END_LIST(API_MS_WIN_CORE_WINRT_STRING_L1_1_0)
+
+DELAY_LOADED_MODULE("api-ms-win-core-winrt-l1-1-0", API_MS_WIN_CORE_WINRT_L1_1_0)
+
+DELAY_LOADED_FUNCTION_BEGIN_LIST(API_MS_WIN_CORE_WINRT_L1_1_0)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_L1_1_0, RoGetActivationFactory)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_L1_1_0, RoUninitialize)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_L1_1_0, RoInitialize)
+DELAY_LOADED_FUNCTION_END_LIST(API_MS_WIN_CORE_WINRT_L1_1_0)
+
+DELAY_LOADED_MODULE("api-ms-win-core-winrt-roparameterizediid-l1-1-0", API_MS_WIN_CORE_WINRT_ROPARAMETERIZEDIID_L1_1_0)
+
+DELAY_LOADED_FUNCTION_BEGIN_LIST(API_MS_WIN_CORE_WINRT_ROPARAMETERIZEDIID_L1_1_0)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_CORE_WINRT_ROPARAMETERIZEDIID_L1_1_0, RoGetParameterizedTypeInstanceIID)
+DELAY_LOADED_FUNCTION_END_LIST(API_MS_WIN_CORE_WINRT_ROPARAMETERIZEDIID_L1_1_0)
+
+DELAY_LOADED_MODULE("api-ms-win-ro-typeresolution-l1-1-0", API_MS_WIN_RO_TYPERESOLUTION_L1_1_0)
+
+DELAY_LOADED_FUNCTION_BEGIN_LIST(API_MS_WIN_RO_TYPERESOLUTION_L1_1_0)
+DELAY_LOADED_FUNCTION_EX(API_MS_WIN_RO_TYPERESOLUTION_L1_1_0, RoParseTypeName)
+DELAY_LOADED_FUNCTION_END_LIST(API_MS_WIN_RO_TYPERESOLUTION_L1_1_0)
+
+HRESULT WINAPI WindowsCreateString(LPCWSTR sourceString, UINT32 length, HSTRING *string)
+{
+    HRESULT hr = S_OK;
+
+    typedef HRESULT WINAPI pFuncWindowsCreateString(LPCWSTR sourceString, UINT32 length, HSTRING *string);
+
+    pFuncWindowsCreateString *pWindowsCreateString = nullptr;
+    IfFailThrow(DelayLoad::API_MS_WIN_CORE_WINRT_STRING_L1_1_0::WindowsCreateString.GetValue(&pWindowsCreateString));
```

Do we expect caller to ever catch the exception? Should we just fail fast?